src/app/models/hris/constants/admin-documents.component.ts

```diff
@@ -0,0 +1,7 @@
+export const Document = [
+  "Criminal Check",
+  "MIE Consent Form",
+  "Qualification Check",
+  "Written approval of offer",
```

Comment:
Written Approval of Offer
```suggestion
  "Written Approval of Offer",
```